src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  father?: Person;
+  mother?: Person;
+};
+
+const PersonLink: React.FC<Props> = ({ person, mother, father }) => {
+  const { slug } = useParams();
+  const isMotherName = person.motherName ? person.motherName : '-';
+  const isFatherName = person.fatherName ? person.fatherName : '-';
```

Comment:
```suggestion
  const motherName = person.motherName ? person.motherName : '-';
  const fatherName = person.fatherName ? person.fatherName : '-';
```
it's not a boolean var to name it starting with `is`

Reply:
fixed